web/app/components/app/text-generate/item/index.spec.tsx

```diff
@@ -0,0 +1,381 @@
+    updateFeedback,
+  }
+})
+
+jest.mock('@/service/debug', () => ({
+  __esModule: true,
+  fetchTextGenerationMessage: jest.fn(),
+}))
+
+jest.mock('@/app/components/base/loading', () => ({
+  __esModule: true,
+  default: () => <div data-testid="loading" />,
+}))
+
+jest.mock('./content-section', () => {
+  const React = require('react')
+  const calls: any[] = []
+  const Component = (props: any) => {
+    calls.push(props)
+    return React.createElement(
+      'div',
+      { 'data-testid': `content-section-${props.taskId ?? 'none'}` },
+      typeof props.content === 'string' ? props.content : 'content',
+    )
+  }
+  return {
+    __esModule: true,
+    default: Component,
+    __calls: calls,
+  }
+})
+
+jest.mock('./meta-section', () => {
+  const React = require('react')
+  const calls: any[] = []
+  const Component = (props: any) => {
+    calls.push(props)
+    return React.createElement(
+      'div',
+      { 'data-testid': `meta-section-${props.messageId ?? 'none'}` },
+      props.showCharCount && React.createElement('span', { 'data-testid': `char-count-${props.messageId ?? 'none'}` }, props.charCount),
+      !props.isInWebApp && !props.isInstalledApp && !props.isResponding && React.createElement(
+        'button',
+        {
+          'data-testid': `open-log-${props.messageId ?? 'none'}`,
+          'disabled': !props.messageId || props.isError,
+          'onClick': props.onOpenLogModal,
+        },
+        'open-log',
+      ),
+      props.moreLikeThis && React.createElement(
+        'button',
+        {
+          'data-testid': `more-like-${props.messageId ?? 'none'}`,
+          'disabled': props.disableMoreLikeThis,
+          'onClick': props.onMoreLikeThis,
+        },
+        'more-like',
+      ),
+      props.canCopy && React.createElement(
+        'button',
+        {
+          'data-testid': `copy-${props.messageId ?? 'none'}`,
+          'disabled': !props.messageId || props.isError,
+          'onClick': props.onCopy,
+        },
+        'copy',
+      ),
+      props.isInWebApp && props.isError && React.createElement(
+        'button',
+        { 'data-testid': `retry-${props.messageId ?? 'none'}`, 'onClick': props.onRetry },
+        'retry',
+      ),
+      props.isInWebApp && !props.isWorkflow && React.createElement(
+        'button',
+        {
+          'data-testid': `save-${props.messageId ?? 'none'}`,
+          'disabled': !props.messageId || props.isError,
+          'onClick': () => props.onSave?.(props.messageId as string),
+        },
+        'save',
+      ),
+      (props.supportFeedback || props.isInWebApp) && !props.isWorkflow && !props.isError && props.messageId && props.onFeedback && React.createElement(
+        'button',
+        { 'data-testid': `feedback-${props.messageId}`, 'onClick': () => props.onFeedback?.({ rating: 'like' }) },
+        'feedback',
+      ),
+    )
+  }
+  return {
+    __esModule: true,
+    default: Component,
+    __calls: calls,
```

Comment:
this pattern is not the standard usage for Jest mock spies; it is recommended to use the built-in capabilities of jest.fn()